packages/changed-elements-react/src/api/VersionCompareManager.ts

```diff
@@ -434,11 +434,14 @@ export class VersionCompareManager {
       if (this.options.wantedModelClasses) {
         wantedModelClasses = this.options.wantedModelClasses;
       }
-
+      let filteredChangeElements = changedElements.slice()
```

Comment:
I would avoid slicing / shallow copying. The changed elements list can be huge for certain iModels. The `_filterIgnoredElementsFromChangesets` already creates the necessary copy.

Reply:
I was wondering about that. I figured it could slow things down for that reason, and I saw we were making a deep copy elsewhere. Ill fix up :D